programmers/K번째-수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, commands) {
+    let answer = [];
+    let answerArray = [];
```

Comment:
1. let -> const
2.변수명은 answerArray 보다는 answers가 적절해보입니다.
```suggestion
   const answer = [];
    let answers = [];
```